test/run-windows-k8s-integration.sh

```diff
@@ -9,9 +9,8 @@
 set -o nounset
 set -o errexit
 
-readonly GCE_PD_DO_DRIVER_BUILD=false
 readonly PKGDIR=${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver
-readonly LOCAL_K8S_DIR=${GOPATH}/src/k8s.io/kubernetes
+readonly k8s_dir=${PKGDIR}/kubernetes
 readonly overlay_name="${GCE_PD_OVERLAY_NAME:-alpha}"
 readonly do_driver_build="${GCE_PD_DO_DRIVER_BUILD:-true}"
```

Comment:
`DO_DRIVER_BUILD` no longer exists

Reply:
I don't see DO_DRIVER_BUILD? It has GCE_PD_DO_DRIVER_BUILD?